scripts/googleAppScripts/SheetToGithubIssue.gs

```diff
@@ -1,100 +1,166 @@
+    var initiativeTwitter = normalizeResponse(formResponse['Initiative X (Twitter)'][0]);
+    var initiativeLinkedIn = normalizeResponse(formResponse['Initiative LinkedIn'][0]);
+    var initiativeYouTube = normalizeResponse(formResponse['Initiative YouTube Page / Channel'][0]);
+    var initiativeDonationLink = normalizeResponse(formResponse['Initiative Donation Link'][0]);
+    var initiativeTags = normalizeResponse(formResponse['Initiative Tags'][0]);
+
+    // Construct the issue body using markdown
+    var issueBody = `### Initiative Name
+${initiativeName}
+
+### Initiative Description
+${initiativeDescription}
+
+### Initiative Details
+${initiativeDetails}
+
+### Initiative Validation Details
+${initiativeValidationDetails}
+
+### Initiative Category
+${initiativeCategory}
+
+### Initiative Main URL
+${initiativeMainURL}
+
+### Initiative Logo
+${initiativeLogo}
+
+### Initiative Website
+${initiativeWebsite}
+
+### Initiative Phone Number
+${initiativePhoneNumber}
+
+### Initiative E-Mail
+${initiativeEmail}
+
+### Initiative WhatsApp
+${initiativeWhatsApp}
+
+### Initiative Telegram
+${initiativeTelegram}
+
+### Initiative Drive
+${initiativeDrive}
+
+### Initiative Form
+${initiativeForm}
+
+### Initiative Document
+${initiativeDocument}
+
+### Initiative Discord
+${initiativeDiscord}
+
+### Initiative Facebook
+${initiativeFacebook}
+
+### Initiative Instagram
+${initiativeInstagram}
+
+### Initiative TikTok
+${initiativeTikTok}
+
+### Initiative X (Twitter)
+${initiativeTwitter}
+
+### Initiative LinkedIn
+${initiativeLinkedIn}
+
+### Initiative YouTube Page / Channel
+${initiativeYouTube}
+
+### Initiative Donation Link
+${initiativeDonationLink}
+
+### Initiative Tags
+${initiativeTags}`;
+
+    var issueTitle = `[NEW-INITIATIVE]: ${initiativeName}`;
 
     // Call the function to create a GitHub issue
     createGithubIssue(issueTitle, issueBody);
 }
 
 function createGithubIssue(title, body) {
-  var scriptProperties = PropertiesService.getScriptProperties();
-  var githubToken = scriptProperties.getProperty('GITHUB_TOKEN');
-  var repo = '4tals/LinksForIsrael';
-  var url = 'https://api.github.com/repos/' + repo + '/issues';
-
-  var payload = {
-    title: title,
-    body: body,
-    labels: ['new-initiative-request']
-  };
-
-  var options = {
-    method: 'post',
-    headers: {
-      Authorization: 'token ' + githubToken,
-      Accept: 'application/vnd.github.v3+json'
-    },
-    contentType: 'application/json',
-    payload: JSON.stringify(payload),
-    muteHttpExceptions: true // This will prevent the script from throwing an exception on HTTP errors
-  };
-
-  try {
-    var response = UrlFetchApp.fetch(url, options);
-    Logger.log(response.getContentText());
-  } catch (error) {
-    Logger.log('Error creating GitHub issue: ' + error.toString());
-  }
+    var personalGithubToken = PropertiesService.getUserProperties().getProperty('GITHUB_TOKEN');
+    var repo = '4tals/LinksForIsrael';
```

Comment:
I would also take this from the user properties, so each developer could execute this script against their fork

Reply:
Why can't the developer just change the value in here when he wants to test it against his fork?
Either way, the developer will need to define the repo of his choosing somewhere in order for this to work.

Reply:
I extracted it, but not sure its needed.

Reply:
T

> Why can't the developer just change the value in here when he wants to test it against his fork? Either way, the developer will need to define the repo of his choosing somewhere in order for this to work.

They can do that, but the file is tracked on git so they could accidentally check in their changes, which is not the end of the world but just creates unnecessary noise.